hadoop-ozone/recon-codegen/src/main/java/org/hadoop/ozone/recon/schema/StatsSchemaDefinition.java

```diff
@@ -54,6 +54,12 @@ public void initializeSchema() throws SQLException {
     }
   }
 
+  @Override
+  public void upgradeSchema(String fromVersion, String toVersion)
+      throws SQLException {
+    // No schema upgrades needed for the stats table.
```

Comment:
Can we put a log that no schema version upgraded for this XYZ table in this version ?